src/hiero_sdk_python/tokens/custom_fixed_fee.py

```
@@ -128,7 +127,9 @@ def set_hbar_amount(self, amount: Hbar) -> "CustomFixedFee":
         self.amount = amount.to_tinybars()
         return self
 
-    def set_denominating_token_id(self, token_id: typing.Optional["TokenId"]) -> "CustomFixedFee":
+    def set_denominating_token_id(
+        self, token_id: typing.Optional["TokenId"]
```

Ruff (0.14.14)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)